zipline/finance/transaction.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, sid, amount, dt, price, order_id, commission=None):
     def __getitem__(self, name):
         return self.__dict__[name]
 
+    def __repr__(self):
```

Comment:
Also, as a heads-up, I think `sid` here is going to change to `asset` in @jbredeche's latest (`sid` originally stood for "security identifier", but in modern zipline it just refers to the unique integer value assigned to every asset; in either case, the use of `sid` here is actually misleading, since the expected value is an `Equity` or `Future`) in https://github.com/quantopian/zipline/pull/1757.  Might be easiest to wait for that to merge (which should be later today) and then update here.